doc/users/event_handling.rst

```diff
@@ -75,6 +75,35 @@ Event name             Class            Description
 'axes_leave_event'     `.LocationEvent` mouse leaves an axes
 ====================== ================ ======================================
 
+.. note::
+   When connecting to 'key_press_event' and 'key_release_event' events,
+   you may encounter inconsistencies between the different user interface
+   toolkits that Matplotlib works with. This is due to inconsistencies/limitations
+   of the user interface toolkit. The following table shows some basic examples of
+   what you may expect to receive as key(s) from the different user interface toolkits,
+   where a comma separates different keys:
+
+   ============== ============================= ============================== ============================= ============================== ==============================
+   Key(s) Pressed WxPython                      Qt                             WebAgg                        Gtk                            Tkinter
+   ============== ============================= ============================== ============================= ============================== ==============================
+   Shift+2        shift, shift+2                shift, "                       shift, "                      shift, "                       shift, "
+   Shift+F1       shift, shift+f1               shift, shift+f1                shift, shift+f1               shift, shift+f1                shift, shift+f1
+   Shift          shift                         shift                          shift                         shift                          shift
+   Control        control                       control                        control                       control                        control
+   Alt            alt                           alt                            alt                           alt                            alt
+   AltGr          Nothing                       Nothing                        alt                           iso_level3_shift               iso_level3_shift
+   CapsLock       caps_lock                     caps_lock                      caps_lock                     caps_lock                      caps_lock
+   A              a                             a                              A                             A                              A
+   a              a                             a                              a                             a                              a
+   Shift+a        shift, A                      shift, A                       shift, A                      shift, A                       shift, A
+   Shift+A        shift, A                      shift, A                       shift, a                      shift, a                       shift, a
```

Comment:
What keyboard has a capital A already? Do you mean with caps lock on?

Reply:
Yes, I mean with caps lock on.